src/Input/InputEvents.cpp

```diff
@@ -83,6 +85,8 @@ namespace InputEvents
   static Mode overlay_mode = MODE_DEFAULT;
 
   static unsigned MenuTimeOut = 0;
+  
+  bool LockScreen = 0;  /* variable used to store lockscreen enabling */
```

Comment:
Unused variable. I don't understand the code comment - it's a "variable", duh, of course it is, that's obvious from the syntax - and it stores "lockscreen enabling", but this sounds like gibberish to my ears. What does that mean?